css/blocks/footer/footer.less

```
@@ -0,0 +1,98 @@
+.footer {
+  background: #212122;
+  padding-top: 30px;
+}
+
+.footer-links-conteiner {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 30px;
+
+  @media @screen-gt-s {
+    justify-content: space-between;
+  }
+}
+
+.footer-social {
+  display: flex;
+  width: 100%;
+  margin-bottom: 30px;
+  box-sizing: border-box;
+  
+  &__icon {
+    width: 36px;
+    height: 36px;
+    display: block;
+    margin-right: 15px;
+    text-indent: -9999px;
+    cursor: pointer;
+    background: no-repeat 50% 50%;
+  }
+
+  @media @screen-gt-s {
+    width: 204px;
```

здесь ты стараешься "подогнать" под макет. На самом деле контейнер для соц.сетей должен иметь ширину 3M и 4L колонок соответственно.
Чтобы явно привязать размеры к сетке можно воспользоваться переменными
```suggestion
    width: @column-width-m * 3 - @column-padding-m * 2;
```

---

Поправил.